tests/imfile-symlink.sh

```diff
@@ -0,0 +1,338 @@
+#!/usr/bin/bash
+
+output="/var/log/messages"
+rsyslogconf="/etc/rsyslog.conf"
+rsyslogconf_bak=`mktemp`
+cp $rsyslogconf $rsyslogconf_bak
+watchdir0="/var/log/test0"
+targetdir0="/var/log/target0"
+reltarget0="../target0"
+watchdir1="/var/log/test1"
+targetdir1="/var/log/target1"
+reltarget1="../target1"
+
+if [ "$EUID" -ne 0 ]; then
```

Comment:
do we have a chance to get a test without root permissions?

Reply:
Well, the test needs permissions to restart rsyslog to properly flush the watches while re-creating test dirs, so not likely. Is it a big problem when we have root test?

Reply:
Yes.  Why not write a test that uses the test framework, which is able to configure and run rsyslog with out redhat/fedora-isms such as running as root, running using systemd, etc.?
For example: https://github.com/rsyslog/rsyslog/blob/master/tests/imfile-wildcards-dirs-multi.sh

Reply:
+1 - we should have tests for this functionality. If it is just a start, that's "biz as usual" inside the testbench.

Reply:
Yes, I am planning to rewrite the test, what is a timeframe for this to make the 8.36 please?